packages/browser-wallet/src/popup/shared/Modal/Modal.tsx

```diff
@@ -79,6 +81,7 @@ export default function Modal<T extends WithOnClick = WithOnClick>({
     bottom = false,
     middle = false,
     stableScrollbarGutter = false,
+    showCloseButton = true,
```

Comment:
```suggestion
    hideCloseButton = false,
```
:upside_down_face: 
Usually we have booleans default to false :thinking: